src/core/server/index.ts

```diff
@@ -74,6 +75,7 @@ export type {
   CoreConfigUsageData,
   CoreEnvironmentUsageData,
   CoreServicesUsageData,
+  ConfigUsageData,
```

Comment:
Just following what we had with `CoreUsageData` which is also marked as internal while exported i'm assuming it was marked as internal since it is only intended to be used by the collector.